content/inferno/time-series/time-series.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Time Series
+author: clone95
+description: This guide aims to show you the Data Science applications you can build for dealing with time series data.
+---
+
+# Index
+
+- [Time Is Money](#Time-Is-Money)
+- [Introduction To Time Series](#Introduction-To-Time-Series)
+- [Time Series Problems Overview](#Time-Series-Problems-Overview)
+    - [Time Series Analysis](#Time-Series-Analysis)
+    - [Time Series Forecasting](#Time-Series-Forecasting)
+    - [Time Series Classification](#Time-Series-Classification)
+    - [Time Series Forecasting as a Classification Problem](#Time-Series-Forecasting-as-a-Classification-Problem)
+    - [The Matrix Profile](#The-Matrix-Profile)
+    - [Automatic Time Series Feature Extraction](#Automatic-Time-Series-Feature-Extraction)
+- [Conclusions](#Conclusions)
+
+
+**Disclaimer!**
+
+This guide is not intended to be exhaustive of everything you can do with time series data, but to offer you a good overview of the possible approaches you can take or things you should check when dealing with them. 
+
+Having that said, let's begin!
+
+## Time Is Money
+
+Whether it is the sale of products, the estimated production of an agricultural field, or the forecast of any activity of a company, time series are an indispensable type of data, which has been used for thousands of years to inform the activities to be done **today**, based on what will happen **tomorrow**. 
```

Comment:
> to inform the activities to be done **today**, based on what will happen **tomorrow**. 

Many of the ideas of time-series comes from predicting/forecasting tomorrow based on the present which is calculated from the past sometimes of present data is absent, how do we fit this idea in the above sentence? Maybe you could restructure the above to incorporate it as well.